platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init

```diff
@@ -33,7 +33,11 @@ case "$1" in
 start)
         echo -n "Setting up board... "
 
+        rmmod i2c-i801
+        rmmod i2c-ismt
         modprobe i2c-dev
+        modprobe i2c-i801
+        modprobe i2c-ismt
```

Comment:
i do not feel the method here is robust since the root cause of reording is not well understood. can you use i2cdetect to find the correct bus number and use the number accordingly?

Reply:
Do you mean I should use i2cdetect to find the bus number and update fancontrol config ?

Reply:
updated in https://github.com/Azure/sonic-buildimage/pull/6889/commits/ba095594ae968bcd6c605146ac8cfc9e53b95292